### PR TITLE
Add Shupapan

### PR DESCRIPTION
**Pull request by takanabe**
Add a new Cask. :)
- Shupapan v.5.9.7

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

**Comment by takanabe** [comment]
I changed my codes with your example :)

File: Casks/shupapan.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'shupapan' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://sunsky3s.s41.xrea.com/files/shupapan_ub.zip'
+  name 'Shupapan'
+  homepage 'http://sunsky3s.s41.xrea.com/shupapan/'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/shupapan.rb]: This needs a `# todo: change license and remove this comment; ':unknown' is a machine-generated placeholder` comment in front. [Example](https://github.com/caskroom/homebrew-cask/blob/6a66da2e01e5d9b7b71921045aaeb810e22ff206/Casks/raw-photo-processor.rb#L9).

**Comment by takanabe** [comment]
I added a comment "# todo: change license and remove this comment; ':unknown' is a machine-generated placeholder comment "

**Pull request merged**